### PR TITLE
feat(cache): add clear cache command

### PR DESCRIPTION
**Pull request by daaanny90**
# Purpose
I faced the problem, that I could not see new project, customer or services while using mite-cli, because those were updated in mite but I was using an old cache.

I find useful to have a command to clear the cache. I don't know the codebase very well, I tried to contribute with the quick solution I build for myself.

Of course, feel free to point me in the right direction and decline the PR if there are other more elegant solutions.

# Description of Change
My quick solution was directly to replace the content of the file were the cache is saved with an empty object.

Then, the next time `mite new` is used, then the cache will be updated with fresh data.

# Todos

- must be cleared if this is really the best solution to achieve it

### CODE REVIEW

File: source/mite.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ program
   .command('stop', 'stop any running counter')
   .command('unlock', 'unlock single time entry')
   .command('users', 'list, filter & search for users')
+  .command('clearcache', 'clear cache of mite data')
```
Comment (Ephigenia) [inline source/mite.js:30]: I’d rather go for just "cache" with a sub-command "clear" or "remove". This leaves it open for other commands like "renew" or "update".

File: source/lib/auto-complete/completions/clearcache.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
+'use strict';
+
+// the completion is identical to the delete
+module.exports = require('./delete.js');
```
[the remark this answers is not in the document]
Comment (daaanny90) [reply source/lib/auto-complete/completions/clearcache.js]: Removed

File: source/mite.js (hunk 1 above)
Comment (daaanny90) [reply source/mite.js]: You're right, I changed command name and description

**Comment by Ephigenia** [comment]
I’ve changed some minor things. Will merge it soon.

There also could be a more specific cache.clear function that could be called from the "customer-new" command or similar to clear the cache automatically when the cached data is changed by a mite-cli command.

**Review by Ephigenia — APPROVED** [review]

**Pull request merged**